includes/class-query.php

```diff
@@ -313,6 +313,37 @@ public function is_activitypub_request() {
 		return false;
 	}
 
+	/**
+	 * Check if content negotiation is allowed for a request.
+	 *
+	 * @return bool True if content negotiation is allowed, false otherwise.
+	 */
+	public function negotiate_content() {
```

Comment:
How about `use_content_negotiation()` to reference the underlying term, or `should_negotiate_content()` to keep with the interrogative convention (like `is_`, `has_`).